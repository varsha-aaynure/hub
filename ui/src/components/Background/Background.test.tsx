import React from 'react';
import { shallow } from 'enzyme';
import Background from '.';

it('should render the background component', () => {
  const component = shallow(<Background />);
  expect(component.debug()).toMatchSnapshot();
});
