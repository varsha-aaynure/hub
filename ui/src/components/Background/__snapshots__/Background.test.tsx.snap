// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the background component 1`] = `
"<div className=\\"hub-background-image\\" style={{...}}>
  <TextContent>
    <Text component=\\"h1\\" className=\\"hub-background-heading\\">
      Welcome to Tekton Hub
    </Text>
    <Text component=\\"h3\\" className=\\"hub-tag-line\\">
      Discover, search and share reusable Tasks and Pipelines
    </Text>
  </TextContent>
</div>"
`;
